curriculum/utils.js

```diff
@@ -36,6 +36,29 @@ const superBlockToOrder = {
   'relational-database': 12
 };
 
+/**
+ * This order is used for i18n instances where a new certification is released
+ * from beta but is not audited, so cannot be reordered (due to the way we
+ * split the map)
+ */
+const superBlockNonAuditedOrder = {
+  'responsive-web-design': 0,
+  'javascript-algorithms-and-data-structures': 1,
+  'front-end-development-libraries': 2,
+  'data-visualization': 3,
+  'back-end-development-and-apis': 4,
+  'quality-assurance': 5,
+  'scientific-computing-with-python': 6,
+  'data-analysis-with-python': 7,
+  'information-security': 8,
+  'machine-learning-with-python': 9,
+  'coding-interview-prep': 10,
+  '2022/responsive-web-design': 11,
+  'relational-database': 12
+};
+
+const auditedLanguages = ['english'];
```

Comment:
Wondering if we should pull this from [the i18n config.](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/config/i18n/all-langs.ts#L42) Then, once we set the new cert as isAudited, the order would be changed automatically and we wouldn't have to also add the language here.

Reply:
Sounds good, I'm fine with it like this.